target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -67,6 +67,12 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"1:lan" "2:lan" "3:lan" "4:lan" "0:wan" "6@eth0"
 		;;
+	buffalo,wsr-1166dhp|\
+	buffalo,wsr-600dhp)
+		ucidef_add_switch "switch0" \
+			"0:lan:1" "1:lan:2" "2:lan:3" "3:lan:4" "4:wan:5" "6@eth0"
```

Comment:
I think it is the same as `"0:lan" "1:lan" "2:lan" "3:lan" "4:wan"` as it's the same order.

Reply:
Indeed, I'll merge the case.